spec/integration/type/file_spec.rb

```diff
@@ -1091,6 +1091,30 @@ def build_path(dir)
           expect(logs).to be_empty
         end
       end
+
+      context "ensure is absent" do
+        let(:expected_checksum) do
```

Comment:
This could have been done with the with_digest_algorithms shared context, but since it's a temporary behavior I'm happier with this approach.